src/Rector/MethodCall/EloquentOrderByToLatestOrOldestRector.php

```diff
@@ -107,4 +165,16 @@ private function convertOrderByToLatest(MethodCall $methodCall): MethodCall
 
         return $methodCall;
     }
+
+
+    /**
+     * @param mixed[] $configuration
+     */
+    public function configure(array $configuration): void
+    {
+        $allowedPatterns = $configuration[self::ALLOWED_PATTERNS] ?? [];
+        Assert::isArray($allowedPatterns);
```

Comment:
I think there should also be an assertion here to check all values in the array are strings.

Reply:
Good call, done!